language/move-prover/spec-lang/tests/sources/pragmas_err.move

```diff
@@ -0,0 +1,22 @@
+module M {
+    spec module {
+        // Should be fine
+        pragma verify = false;
+
+        // Should produce error that bar is not valid (not defined)
+        pragma bar = true;
+
+        // Should be ok (we do not check types of values)
+        pragma verify = a_valid_id;
```

Comment:
It seems that we allow `pragma verify = false;` and `pragma verify = a_valid_id;` to appear together in a spec block. Maybe it would be less confusing if we generate an error saying something like "pragma verify has conflicting types".

Reply:
Reading the code again I see that we might not be able to check if they are of the same type since we don't know the type of `a_valid_id`, which could be a boolean as well.

Reply:
Pragmas aren't type checked in general. The only thing we check is if the name is a known pragma and valid in the given context. Adding pragma type checking would be an independent, larger change than this one.

Reply:
Another thing: this only allows the pragma value to be one identifier right? What if we want to say some function is called only from both f and g?

Reply:
That won't be possible for now. We can add it later, but perhaps the full Move-based solution might be earlier here than expected.